.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ version: 2
 job_defaults_var: &job_defaults
   working_directory: ~/workspace/app
   docker:
-    - image: circleci/node:8-browsers
+    - image: circleci/node:11.4.0-browsers
```

Comment:
better `circleci/node:12.0.0-browsers` then. Node 12 supports V8 7.4 and it's faster than V8 7.1

Reply:
done